cpp/include/raft/spatial/knn/detail/ann_utils.cuh

```diff
@@ -383,6 +317,8 @@ __global__ void map_along_rows_kernel(
  * @brief Map a binary function over a matrix and a vector element-wise, broadcasting the vector
  * values along rows: `m[i, j] = op(m[i,j], v[i])`
  *
+ * @todo(lsugy): replace with matrix_vector_op
```

Comment:
Can we add a GitHub issue for this and reference it here just to make sure we are tracking it?

Reply:
Oops I didn't mean to leave that comment here, but I already have a WIP PR for this: #911